src/database/migrations/20230401145629-add-column-pasword-status.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Users', 'PasswordStatus', {
```

Comment:
1. why is this is plural? are other fields in plural?
2. isn't this a boolean? then it is named wrong, it should be passwordExpired